auth0/src/test/java/com/auth0/android/authentication/storage/SecureCredentialsManagerTest.kt

```diff
@@ -136,15 +136,15 @@ public class SecureCredentialsManagerTest {
         MatcherAssert.assertThat(storedCredentials.refreshToken, Is.`is`("refreshToken"))
         MatcherAssert.assertThat(storedCredentials.type, Is.`is`("type"))
         MatcherAssert.assertThat(storedCredentials.expiresAt, Is.`is`(Matchers.notNullValue()))
-        MatcherAssert.assertThat(storedCredentials.expiresAt!!.time, Is.`is`(sharedExpirationTime))
+        MatcherAssert.assertThat(storedCredentials.expiresAt.time, Is.`is`(sharedExpirationTime))
         MatcherAssert.assertThat(storedCredentials.scope, Is.`is`("scope"))
     }
 
     @Test
     public fun shouldSaveRefreshableCredentialsUsingAccessTokenExpForCacheExpirationInStorage() {
         val accessTokenExpirationTime = CredentialsMock.ONE_HOUR_AHEAD_MS
         val credentials: Credentials = CredentialsMock(
-            null,
+            "",
```

Comment:
this case will never happen again, as Credentials will always have these properties passed as empty here. I'd remove this test case.